sessions/js-objects-and-arrays/objects-and-arrays/index.js

```diff
@@ -0,0 +1,122 @@
+import checkConditions from './helper';
+
+console.clear();
+
+const people = [
+  {
+    name: 'Homer',
+    age: 39,
+    parents: [{mothersName: 'Mona', fathersName: 'Abe'}],
+    pet: [{animal: 'dog', name: "Santa's Little Helper"}],
+    friend: [{name: 'Moe Szyslak', work: 'owner of Moes Tavern'}],
```

Comment:
I wouldn't show any data structures that only make sense for the exercise but not otherwise.
Here single objects would be enough.